Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM ubuntu:20.10
+FROM fedora:rawhide
```

Comment:
Is rawhide necessary since you bootstrap almost everything?

Reply:
Hmm, it seems so, on fedora:latest it fails due to outdated meson: https://github.com/jf2048/gtk-rs-core/runs/6371974889#step:3:1407

Reply:
Could install meson manually and avoid rawhide

Reply:
The image is only built manually, I wouldn't bother with that now